index.js

```diff
@@ -12,6 +12,11 @@ export default {
     if (options.action && options.action.color) {
       options.action.color = processColor(options.action.color);
     }
+
+     if(!options.tintColor) {
```

Comment:
Oh I just called it tintColor cause that's what it's called on all the other react native components.